build.sbt

```diff
@@ -75,8 +75,8 @@ lazy val commonSettings =
     fork in test := true
   )
 
-addCommandAlias("build", ";core/compile;core/test;tests/test")
-addCommandAlias("validate", ";scalastyle;build")
+addCommandAlias("build", ";compile;test")
```

Comment:
Someone who understands scoverage, please check that this change is fine. (There are no tests in `core` though.)